model/src/main/kotlin/Package.kt

```diff
@@ -141,6 +141,24 @@ data class Package(
         )
     }
 
+    /**
+     * Check if this package contains any erroneous data.
+     */
+    fun collectErrors(): List<OrtIssue> {
+        val issues = mutableListOf<OrtIssue>()
```

Comment:
Should we also use the more compact function expression syntax with `= mutableListOf<OrtIssue>().also { issues ->` here?

Reply:
Done, used `map` to make it even more compact.